python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py

```diff
@@ -2168,6 +2168,12 @@ def convert_sum(node, **kwargs):
         )
     return [node]
 
+@mx_op.register("sum_axis")
+def convert_sum_axis(node, **kwargs):
+    """Map MXNet's sum_axis operator.
+       sum_axis is equivalent to sum in MXNet
+    """
+    return convert_sum(node, **kwargs)
```

Comment:
Instead of wrapping convert_sum() here, couldn't we just add the decorator to register "sum_axis" to convert_sum() ?

Reply:
Registered it under convert_sum. Thanks for the good point!